CMakeLists.txt

```diff
@@ -134,6 +134,13 @@ if(
   )
 endif()
 
+# ---- RISC-V V-extension auto-detect ----
```

Comment:
We don't want to do that at the CMake level because we do not assume that our users rely on CMake.

Instead, the code should rely on the macro `SIMDJSON_IS_RVV`. Please see:

https://github.com/simdjson/simdjson/blob/89f2b634df9fe9aa2b08f00a739aa6cc11418dbd/include/simdjson/portability.h#L58C13-L58C28